src/main/java/wooteco/subway/line/LineController.java

```diff
@@ -0,0 +1,68 @@
+package wooteco.subway.line;
+
+import java.net.URI;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import wooteco.subway.line.dto.LineRequest;
+import wooteco.subway.line.dto.LineResponse;
+
+@RestController
+@RequestMapping("/lines")
+public class LineController {
+
+    private final LineDao lineDao;
+
+    public LineController(LineDao lineDao) {
+        this.lineDao = lineDao;
+    }
+
+    @PostMapping
+    public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
+        Line line = new Line(lineRequest.getName(), lineRequest.getColor());
+        Line newLine = lineDao.save(line);
+        LineResponse lineResponse = new LineResponse(newLine.getId(), newLine.getName(),
+            newLine.getColor());
```

Comment:
```suggestion
        LineResponse lineResponse = new LineResponse(newLine);
```

이렇게 한다면 이전 대비 어떤 장단점이 있을까요?
(리뷰이들의 코드가 모두 동일한데... 기본 제공 코드를 바꾸면 큰일난다고 생각하신 걸까요? 😅)

Reply:
바로 수정하겠습니다.